pkg/controller/node.go

```diff
@@ -38,29 +38,36 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-// ReconcileAzDriverNode reconciles AzDriverNode
-type ReconcileAzDriverNode struct {
+// ReconcileNode reconciles AzDriverNode
+type ReconcileNode struct {
 	*SharedState
 	logger logr.Logger
 }
 
 // Implement reconcile.Reconciler so the controller can reconcile objects
-var _ reconcile.Reconciler = &ReconcileAzDriverNode{}
+var _ reconcile.Reconciler = &ReconcileNode{}
 
-func (r *ReconcileAzDriverNode) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
+func (r *ReconcileNode) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	if !r.isRecoveryComplete() {
 		return reconcile.Result{Requeue: true}, nil
 	}
 
 	n := &corev1.Node{}
 	err := r.cachedClient.Get(ctx, request.NamespacedName, n)
 
+	logger := r.logger.WithValues(consts.NodeNameLabel, request.Name)
```

Comment:
Try to avoid inserting statements between a function that returns an error and the `if` statement that checks the result. The declaration and assignment of `logger` should be moved above line 55.